### ISSUE TITLE
how do I seed fasthash ?

### ISSUE THREAD

**Issue by lucasart** [ZilongTan/fast-hash#1]
I couldn't find any information in the readme regarding seeding. Can I use any seed, even zero ? In practice, I just want a reproductible hash function,  so the seed may as well be hardcoded.

**Comment by ztanml** [comment]
Yes, you can use any hardcoded integer number like 0xdeadbeef as the seed.
